disk-utils/mkswap.8.adoc

```diff
@@ -73,6 +73,9 @@ generate a new time-based UUID
 *-e*, *--endianness* _ENDIANNESS_::
 Specify the _ENDIANNESS_ to use, valid arguments are *native*, *little* or *big*. The default is *native*.
 
+*-o*, *--offset* _offset_::
+Specify the _offset_ to write the swap header to.
```

Comment:
Pedantic note, it's where the swap area starts ;-) The header is `start+SIGNATURE_OFFSET`.